doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -8,7 +8,7 @@ functions can be called directly from Julia without any "glue" code, code genera
 [`@ccall`](@ref) macro (or the less convenient [`ccall`](@ref) syntax, see the [`ccall` syntax section](@ref ccall-interface)).
 
 The code to be called must be available as a shared library. Most C and Fortran libraries ship
-compiled as shared libraries already, but if you are compiling the code yourself using GCC (or
+compiled code as shared libraries already, but if you are compiling the code yourself using GCC (or
```

Comment:
I don't think this is quite right (or is at least more confusing, IMO). The sentence as originally written is saying that most C and Fortran libraries are themselves compiled as shared libraries. This addition makes it sound like the libraries come packaged with additional compiled code, which may or may not be true, but is irrelevant either way.
```suggestion
compiled as shared libraries already, but if you are compiling the code yourself using GCC (or
```

Reply:
I see. How about “Most C and Fortran libraries are already shipped as shared libraries , but if you are …..”